### DISCUSSION TITLE
Mail Image Stopped Working

### DISCUSSION THREAD

**Discussion by cabr0ne** [moralmunky/Home-Assistant-Mail-And-Packages#395] [Q&A] [answered]
Hi All,
I had my mail image working as of a few days ago but now it is not. I have re-configured everything and have been receiving this error:

"end of the stream or a document separator is expected at line 11, column 1:
    - alias: "Update USPS Mail Camer ... 
    ^"
    
    Here is the config in configuration.yaml

![Capture](https://user-images.githubusercontent.com/78617409/110147887-073b7880-7d91-11eb-9818-e5cc4d16d059.PNG)

Can someone please assist?

Thanks!

**Comment by firstof9** [comment, marked as answer]
Indent the `- alias: "Update USPS Mail Camer...` line.
It should line up with `trigger` like so:

```yaml
- alias: "Update USPS Mail Camera"
  trigger:
    - platform: state
```

**Reply by cabr0ne** [reply]
![Capture](https://user-images.githubusercontent.com/78617409/110166102-b6d01500-7da8-11eb-9d7a-040d1a4a4369.PNG)


Still getting the same error even with it like this

**Reply by firstof9** [reply]
Also place it in your automations.yaml file.
